lms/static/js/student_account/views/FormView.js

```diff
@@ -172,7 +172,7 @@
 
                         if (key) {
                             test = this.validate(elements[i]);
-                            if (test.isValid) {
+                            if (test.isValid || $el.attr('id') === 'login-email') {
```

Comment:
Oh, yes. I only know about the "logistration" (login + register) page as people call it. You should focus on the new one only and although I don't know whether the old one did check or not, since the new one does that's all that really matters going forward.